pkg/types/service_offering.go

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2018 The Service Manager Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package types
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+// ServiceOffering defines the data of a service offering.
+type ServiceOffering struct {
```

Comment:
Check whether we can use directly the type from Service Manager and redefine it here by adding the methods below for being table printed object. Not sure about this